project.config.json

```diff
@@ -7,6 +7,6 @@
 		"newFeature": true
 	},
 	"appid": "wx13a399684987dfa0",
-	"projectname": "TPCC-YEC-Soccer-Team-Mini-Program",
+	"projectname": "homepagem",
```

Comment:
Why do we need to change the project name?

Reply:
I think Xinyi changed this accidentally, will change it back